pkg/pillar/kubeapi/storageutils.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package kubeapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	longhornDevPath = "/dev/longhorn"
+)
+
+func deviceMajor(stat syscall.Stat_t) int32 {
+	return int32((stat.Rdev >> 8) & 0xfff)
```

Comment:
https://pkg.go.dev/golang.org/x/sys/unix#Major

Reply:
Switched over to using that in latest push.